### DISCUSSION TITLE
How to make a backend with Fooocus

### DISCUSSION THREAD

**Discussion by Makxzod** [lllyasviel/Fooocus#1963] [Q&A]
Hi, i am very interested in making an express backend that uses fooocus to generate images and then send a link or b64_json back to the client. Is this possible? Have anyone done something similar?

**Comment by thiner** [comment]
check project `fooocus-api`.

**Comment by mashb1t** [comment]
You can either use the included Gradio API (Websockets, sends also events for progress of rendering incl- percentages) or the REST-API project https://github.com/konieshadow/Fooocus-API referenced by @thiner.
